lib/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Request, Response, CookieOptions } from 'express';
+import { Request, Response, CookieOptions } from './express/mock-express';
```

Comment:
https://github.com/eugef/node-mocks-http/blob/master/lib/express/mock-express.js doesn't export `Request, Response, CookieOptions` types. Are you sure this is working?